ably.d.ts

```diff
@@ -210,6 +210,14 @@ declare namespace Types {
      */
     useBinaryProtocol?: boolean;
 
+    /**
+     * Override the url used by the realtime client to check if the internet is available
+     * for private networks where access to the public internet not available.
```

Comment:
I don't think this comment is very clear, since if a client is in a private network with no access to the public Internet, then why would the user configure `internetAvailabilityUrl` when they know the Internet is not reachable?

There are two use cases here:

1. a user who wants to use an "is the Internet reachable" URL under their own domain (e.g. `https://internet-up.acme.com`)
2. a user who knows their clients can't reach the Internet, so they want to disable the check completely

Perhaps these should be catered for by different client options?

Reply:
OK I'll look into adding an option for disabling the check completely.

As for the comment being unclear, I'm not entirely sure what you're objecting to sorry, if I changed it to "check if the 
_network_ is available" would that make it clear?

Reply:
> As for the comment being unclear, I'm not entirely sure what you're objecting to sorry, if I changed it to "check if the
network is available" would that make it clear?

I think just remove the "for private networks where access to the public internet not available.", since I don't believe this option relates to that use case.

Reply:
I've removed that part of the comment and added a `disableConnectivityCheck` client option to skip the check entirely.

Reply:
Thanks 👍